src/main/java/com/iota/iri/conf/ProtocolConfig.java

```diff
@@ -0,0 +1,37 @@
+package com.iota.iri.conf;
+
+public interface ProtocolConfig extends Config {
+
+    int getMwm();
+
+    int getTransactionPacketSize();
+
+    int getRequestHashSize();
+
+    double getpReplyRandomTip();
```

Comment:
`pReply` should be `PReply

Reply:
Yup,

Per my previous comment, this mistake also breaks INI parsing...
Thanks!

Reply:
I rushed to accept your comment apparently (it just made sense)...

It appears that the correct convention that Jackson accepts is `getpReplyRandomTip()`.
I will revert the change now.